include/arc_utilities/vector_math.hpp

```diff
@@ -0,0 +1,206 @@
+#ifndef ARC_UTILITIES_VECTOR_MATH_HPP
```

Comment:
This file has both `std::pair` and `std::vector` math. Maybe name this `stl_math.hpp`?